requirements.txt

```
@@ -1 +1,2 @@
-Django~=4.0
\ No newline at end of file
+Django=4.0.4
+django-bootstrap5=21.3
```

```suggestion
Django~=4.0.4
django-bootstrap5~=21.3
```
このままだと`pip install -r requirements.txt`時にエラー起きるので、上のように変更お願いします。バージョン固定にしたい場合は~=ではなく、==にしてください。

---

ご指摘ありがとうございます！変更します